src/Network/Receive/ServerType0.pm

```diff
@@ -145,7 +145,7 @@ sub new {
 		'00DB' => ['chat_users'],
 		'00DC' => ['chat_user_join', 'v Z24', [qw(num_users user)]],
 		'00DD' => ['chat_user_leave', 'v Z24 C', [qw(num_users user flag)]],
-		'00DF' => ['chat_modified', 'x2 a4 a4 v2 C a*', [qw(ownerID ID limit num_users public title)]],
+		'00DF' => ['chat_modified', 'v a4 a4 v2 C', [qw(len ownerID ID limit num_users public)]], # -1
```

Comment:
It's better practice to put all of the available fields into the unpack string here, if possible. In addition to being more clear and concise, it also helps people implementing their own packet handlers (via `packet/*` and `packet_pre/*` hooks). Why remove `title`?